packages/typespec-ts/src/modular/helpers/operationHelpers.ts

```diff
@@ -943,3 +943,15 @@ export function hasPagingOperation(codeModel: ModularCodeModel) {
     )
   );
 }
+
+function getAllProperties(type: Type): Property[] {
+  const properties: Property[] = [];
```

Comment:
nit: not sure this is caused the duplicated properties issues when discriminator property exists both parent and child model. see below cadl-ranch cases:

https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-specs/http/type/model/inheritance/single-discriminator/main.tsp#L12-L25C1

Reply:
Yeah, I found that, when I was trying to enable cadl ranch test cases for model inheritance in modular.